components-core/src/main/java/com/airwallex/android/core/data/AirwallexCVCParam.kt

```diff
@@ -0,0 +1,12 @@
+package com.airwallex.android.core.data
+
+import androidx.activity.ComponentActivity
+import com.airwallex.android.core.AirwallexSession
+import com.airwallex.android.core.model.PaymentMethod
+
+data class AirwallexCVCParam(
```

Comment:
I feel the naming can be more generic, for example, after we also support Google Pay recurring mode so we can call it `AirwallexCheckoutParam` (also it matches AirwallexCheckoutActivity).